### ISSUE TITLE
Decorrelation in the new dust models

### ISSUE THREAD

**Issue by myra-norton** [galsci/pysm#123]
I'm working with @brandonshensley to quantify decorrelation in the new models. Here is a first notebook + plot: https://gist.github.com/myra-norton/8edcde151758a5898b0653aadc037d84

**Comment by giuspugl** [comment]
Thanks a lot @myra-norton this is really great! 
It's not immediate to get a sense of why we do see a decrease in the decorrelation at ell>100 ? Do we expect small scales injected w/ poltens to behave a freq. decorrelation ? 
Also are those results consistent w/ planck latest constraints ?

**Comment by brandonshensley** [comment]
I've made an updated version of this plot, see [linked notebook](https://gist.github.com/brandonshensley/f72732fe0983f285196a4f6194e2dec9).

**Comment by zonca** [comment]
@brandonshensley what about adding this analysis to the docs?

at the top of the Dust section at https://pysm3.readthedocs.io/en/latest/models.html#dust I can put a link to this.
If we decide to proceed with this, we will need more text inside the notebook that explains what it is doing and exact link to section of the original paper.

The notebook can then directly be added to the docs.

**Comment by brandonshensley** [comment]
@zonca that's a great idea! Let's wait until after internal review and convergence to a final version, but it would be a good addition to that page. I can expand the text.

**Comment by brandonshensley** [comment]
Good news: I redid the plot with anafast and got an identical result. Notebook [here](https://gist.github.com/brandonshensley/3aeb26051a62aab0d8a081370de8229c).

**Comment by zonca** [comment]
@brandonshensley does this go into the paper? If so you could just add it to the `pysm_methods_paper` repo

**Comment by brandonshensley** [comment]
> @brandonshensley does this go into the paper? If so you could just add it to the `pysm_methods_paper` repo

Yes, will do once @1cosmologist and @kennykinglau have had a look to be sure the methodology agrees with theirs.

**Comment by 1cosmologist** [comment]
@brandonshensley One question: Should you bin R_ell (where R_ell is computed for each ell) or compute R_ell from Cls that are binned over a single big bin? Ratio of averages vs average of ratios.

**Comment by brandonshensley** [comment]
> @brandonshensley One question: Should you bin R_ell (where R_ell is computed for each ell) or compute R_ell from Cls that are binned over a single big bin? Ratio of averages vs average of ratios.

I thought it made more sense to bin the power spectra then compute R_ell last (i.e., we are reporting the correlation between the BB power averaged over a broad ell bin vs the average level of correlation over that bin). Since R_ell is generally pretty flat with ell, I don't think it matters much.

**Comment by 1cosmologist** [comment]
Fair enough.

**Comment by zonca** [comment]
This issue is good for reference but does not need to remain open.
